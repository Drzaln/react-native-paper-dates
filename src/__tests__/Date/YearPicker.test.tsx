import { render } from '@testing-library/react-native'
import YearPicker from '../../Date/YearPicker'

it('renders YearPicker', () => {
  const { toJSON } = render(
    <YearPicker
      selectedYear={2024}
      selectingYear={true}
      onPressYear={() => null}
      startYear={1800}
      endYear={2200}
    />
  )
  expect(toJSON()).toMatchSnapshot()
})
