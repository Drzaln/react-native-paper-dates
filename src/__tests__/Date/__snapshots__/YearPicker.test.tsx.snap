// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders YearPicker 1`] = `null`;
